docker-build/docs/images.md

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
This file is currently at `docker-build/docs/images.md`, but it should be moved to `docs/images.md` (to match the link/reference in the main Readme).